src/planscape/plan/models.py

```diff
@@ -83,6 +83,11 @@ class ProjectArea(models.Model):
     ProjectAreas are associated with one Project. Each ProjectArea has geometries representing 
```

Comment:
Please update the comment ("associated with one User, the owner, and one Project.")